Podfile

```diff
@@ -3,15 +3,29 @@ xcodeproj 'malaria-ios.xcodeproj'
 
 platform :ios, '8.0'
 use_frameworks!
-link_with 'malaria-ios', 'malaria-iosTests'
 
-pod 'Alamofire', '~> 3.1.4'
-pod 'SwiftyJSON', '~> 2.3.1'
-pod 'Charts', '~> 2.1.6'
-pod 'CVCalendar', '~> 1.2.8'
+def testing_pods
```

Comment:
As far as I can browse the code, there is no framework required for malaria_tests. Am I right?